### PR TITLE
[ROS_MELODIC] Add Mellinger's and Silano et. al controllers

### PR DESCRIPTION
**Pull request by gsilano**
The new version aims to add two news controller algorithms: the Mellinger [1] and Silano et. al [2]. This branch concerns ROS Melodic version.

[1] D. Mellinger and V. Kumar, "Minimum snap trajectory generation and control for quadrotors," 2011 IEEE International Conference on Robotics and Automation, Shanghai, 2011, pp. 2520-2525. DOI: [10.1109/ICRA.2011.5980409](https://doi.org/10.1109/ICRA.2011.5980409)
[2] G. Silano, P. Oppido and L. Iannelli, "Software-in-the-loop simulation for improving flight control system design: a quadrotor case study," 2019 IEEE International Conference on Systems, Man and Cybernetics (SMC), Bari, Italy, 2019, pp. 466-471. DOI: [10.1109/SMC.2019.8914154](https://doi.org/10.1109/SMC.2019.8914154)

### CODE REVIEW

File: rotors_gazebo/CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,10 @@ Changelog for package rotors_gazebo
 
 6.0.2 (20XX-XX-XX)
```
Comment (gsilano) [inline rotors_gazebo/CHANGELOG.rst]: Remember to update this field before merging the changes in the master

**Comment by marco-marino95** [comment]
I followed this guide, however when I run the " roslaunch rotors_gazebo crazyflie2_hovering_example.launch " command the crazyFlie gets up after 5 seconds of 1 meter.
How do I go about giving him a position that I want?

**Comment by gsilano** [comment]
@marco-marino95, please open an issue related to your problem. This conversation concerns the PR that has not yet been integrated into the master.

**Comment by senceryazici** [comment]
Hi,
Are there any estimations when this Pull Request will be ready to merge ?

Thanks

**Comment by gsilano** [comment]
Hi @senceryazici! No, sorry. I'm still working on it and I don't know how long it will take.

**Pull request merged**